lang/step.md

```diff
@@ -28,7 +28,9 @@ impl<M: Memory> Machine<M> {
         } else {
             // Bump up PC, evaluate this statement.
             let stmt = block.statements[next_stmt];
-            self.cur_frame_mut().next_stmt = next_stmt + 1;
+            self.mutate_cur_frame(|frame| {
+                frame.next.1 = next_stmt + 1;
```

Comment:
I'll refactor this a bit more after merging, I don't like the assignment to an anonymous tuple field here.

Reply:
Or maybe you can do that... basically replace `next` by `next_block` and `next_stmt`.